web/package.json

```diff
@@ -13,12 +13,13 @@
   },
   "dependencies": {
     "@chakra-ui/next-js": "^2.0.1",
-    "@chakra-ui/react": "^2.5.1",
+    "@chakra-ui/react": "^2.6.0",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
     "@fontsource/inter": "^4.5.15",
     "@hookform/resolvers": "^2.9.11",
     "@next/font": "13.1.6",
+    "@ramonak/react-progress-bar": "^5.0.3",
```

Comment:
what is this ?

Reply:
i used that for importing react progress bar

Reply:
where are you using this progress bar ?

Reply:
i dint use it

Reply:
![image](https://user-images.githubusercontent.com/66205494/235980104-9b051060-252f-4ed3-99f2-8d4a0a27173e.png)

you only updated chakra ui and added the modal why too much file changes ?

Reply:
> i dint use it

pinne enthina nee install akie ?